config/Sage/config.txt

```diff
@@ -551,3 +551,6 @@ VOTE_AUTOTRANSFER_INTERVAL 18000
 
 ## Ghost role cooldown time after death (In deciseconds)
 GHOST_ROLE_COOLDOWN 3000
+
+## Enable/disable roundstart station traits
+#STATION_TRAITS
```

Comment:
These should be enabled by default to match the server config.

Reply:
This is similar to #3766
It is supposed to be disabled in repo so people don't get traits while debugging stuff but enabled on the live server

Reply:
It should be disabled on the repo so that dev builds don't use station traits.